src/Edit.tsx

```diff
@@ -313,7 +306,13 @@ const AppNoError = ({
               }}
               initialMode="tree"
               // Note: these offsets are rather arbitrary.
-              initialPosition={{ x: canvasDimensions.width - 100, y: 15 }}
+              initialPosition={{ x: canvasDimensions.width - 410, y: 10 }}
```

Comment:
This concerns me a little. I think we do have an issue somewhere about removing this. But until then, perhaps a comment about what these arbitrary numbers are intended to achieve?

Reply:
They've been there all along, of course, and I'm just changing the offset here. Anyway, this is just a hack to get the initial position of the window in the bounds of the canvas.

This is fixed slightly more robustly in an upcoming PR.